test/v2/services/CreateForeignPropertyPeriodSummaryServiceSpec.scala

```diff
@@ -165,6 +165,7 @@ class CreateForeignPropertyPeriodSummaryServiceSpec extends UnitSpec {
       val input = Seq(
```

Comment:
Also please add `"DUPLICATE_COUNTRY_CODE" -> RuleDuplicateCountryCode`
which was just added to the tech spec. (May need new error for this.)